src/QirRuntime/lib/QIR/bridge-qis.ll

```diff
@@ -60,6 +60,8 @@ declare void @quantum__qis__y__ctl(%struct.QirArray*, %class.QUBIT*)
 declare void @quantum__qis__z__body(%class.QUBIT*)
 declare void @quantum__qis__z__ctl(%struct.QirArray*, %class.QUBIT*)
 
+declare void @quantum__qis__message__body(%"struct.QirString"* %str)

```

Comment:
FYI: this is another situation when quotes aren't necessary

Reply:
OK, when touching the lines I will keep that in mind.